public/app/features/explore/Logs.tsx

```diff
@@ -123,6 +135,17 @@ export class UnthemedLogs extends PureComponent<Props, State> {
     this.cancelFlippingTimer = setTimeout(() => this.setState({ isFlipping: false }), 1000);
   };
 
+  onEscapeNewlines = () => {
+    this.setState({ isEscaping: true });
```

Comment:
Not sure I get why this is here. Does it take such a long time to rerender when escaping that we want to have essentially a loading state? If so does that mean it blocks all other actions?

Reply:
Not really. It escapes the lines right away. I just liked that it visually showed you that something is happening. The same way we are doing it in fliping order. But the loading logic could be removed.

Reply:
But yeah, you are right. Too much code with a little/no effect. I have removed it.